pos_accented_search/static/src/js/db.js

```diff
@@ -0,0 +1,25 @@
+odoo.define('pos_accented_search', function (require) {
+
+    "use strict";
+    var db = require("point_of_sale.DB");
+    db.include({
+
+        normalize_characters: function (product) {
+            return product.normalize("NFD")
+                .replace(/[\u0300-\u036f]/g, "")
```

Comment:
maybe you could add a little comment to explain what are there codes ?

Reply:
Done! see the latest commit.